content/influxdb/cloud/write-data/migrate-data/migrate-cloud-to-cloud.md

```diff
@@ -385,17 +392,30 @@ task will fail.
   a smaller interval. Each batch will then query less data and take less time
   to return results.
 
-### Migrate to InfluxDB Cloud Serverless powered by IOx
+### Batch size is too large
+
+If your batch size is too large, the task will return an error similar to:
+
+```
+internal error: error calling function "metadata" @97:1-97:11: error calling function "findRecord" @67:32-67:69: wrong number of fields
+```
+
+**Possible solutions**:
+- Update the `migration.batchInterval` setting in your migration task to use
+  a smaller interval. Each batch will then query less data.
```

Comment:
~~~suggestion
If your batch size is too large, the task returns an error similar to the following:

```
internal error: error calling function "metadata" @97:1-97:11: error calling function "findRecord" @67:32-67:69: wrong number of fields
```

**Possible solutions**:
- Update the `migration.batchInterval` setting in your migration task to use
  a smaller interval and retrieve less data per batch.
~~~